packages/zoe/src/contracts/loan/types-ambient.js

```diff
@@ -43,7 +43,7 @@
 
 /**
  * @typedef LenderSeatProperty
- * @property {ZCFSeat} lenderSeat
+ * @property {import('../../types-index').ZCFSeat} lenderSeat
```

Comment:
`ZCFSeat` is used several times. Consider importing it once.

Reply:
This way was something the IDE did for me. I'm going to leave it done if that's ok